examples/preview/theme/front-page.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { usePosts, useGeneralSettings } from '@wpengine/headless';
+import { CTA, Header, Footer, Hero, Posts } from '../components';
+import styles from '../scss/theme/front-page.module.scss';
+
+export default function FrontPage(): JSX.Element {
+  const posts = usePosts();
+  const settings = useGeneralSettings();
+
+  return (
+    <>
+      <Header title={settings?.title} description={settings?.description} />
+      <main className="content">
+        <Hero
+          title="Get Started with Headless"
+          buttonText="Developer Docs"
+          buttonURL="https://developers.wpengine.com/"
+          button2Text="Headless on GitHub"
+          button2URL="https://github.com/wpengine/headless-framework"
+          bgImage="/images/headless_hero_background.jpg"
+          id={styles.home_hero}>
+          <p>
+            WP&nbsp;Engine’s Headless WordPress Framework includes this starter
+            theme, the{' '}
```

Comment:
"includes this example project".

I want to avoid calling it a starter theme for now

Reply:
Makes sense! Changed in https://github.com/wpengine/headless-framework/pull/109/commits/63df63cb4c37f4e7113b88f695d91feb413bde82.